src/main/app/case/from-api-format.ts

```diff
@@ -44,6 +44,17 @@ const fields: FromApiConverters = {
       placementOrderDate: fromApiDate(item.value.placementOrderDate),
     })),
   }),
+  siblings: data => ({
+    siblings: (data.siblings || []).map(sibling => ({
+      ...sibling.value,
+      siblingPlacementOrders: ((sibling.value.siblingPlacementOrders || []) as ListValue<PlacementOrder>[]).map(
+        placementOrder => ({
+          ...placementOrder.value,
+          placementOrderDate: fromApiDate(placementOrder.value.placementOrderDate),
```

Comment:
I think this line is not required, as `placementOrderDate` is not in scope for siblings